### DISCUSSION TITLE
Can anyone explain me what is <dl> exactly and why they use it in top solutions?

### DISCUSSION THREAD

**Discussion by thoughtlessnerd** [cssbattle/forum#36] [Q&A] [answered]
here is the top solution for target #71 elphant:

`<dl><dl><dd><dt><p><style>body{margin:-32;padding:68 118;--:9q,#0000 42Q,#998235}* * *{height:180;border-radius:2IN;--a:0}dd{--:at 61%0,#0000,10q,#998235 0 21q,#1A4341 0;--a:95q/63q}dt{--:at bottom,#0000 5CH,#FFF 0 64Q,#0000;--a:70px}p{margin:-72 46}*{margin:12;--:9Q,#0B2429 42Q,#1A4341;background:radial-gradient(4q var(--)42.7Q)0 var(--a,/50%`

please explain me someone.

**Comment by iamvdo** [comment, marked as answer]
`<dl>` is a *definition list*, and one char shorter than `<div>`. That's it.

**Reply by jbarrantes** [reply]
What would be the advantage vs just using `<p>` that is even shorter?

**Reply by aaronze** [reply]
The `<p>` tag comes in built with a couple of properties that could be undesirable, like inline-block, and it also can't be nested, i.e:
`<p><p>` is actually rendered as: `<p></p><p></p>`
whereas
`<dl><dl>` is actually rendered as `<dl><dl></dl></dl>`
